app/src/test/java/src/RandomShapeTest.java

```diff
@@ -0,0 +1,27 @@
+package src;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class RandomShapeTest {
+
+    @Test
+    public void itShouldGetShape() {
```

Comment:
The name of the test method is not according to the class agreement on naming.

Reply:
Thanks ray. I changed it